neo4j-jdbc-translator/impl/src/main/java/org/neo4j/jdbc/translator/impl/SqlToCypher.java

```diff
@@ -347,10 +408,32 @@ private ResultStatement statement(Select<?> incoming) {
 				return Cypher.returning(resultColumnsSupplier.get()).build();
 			}
 
-			OngoingReadingWithoutWhere m1 = Cypher
-				.match(x.$from().stream().flatMap(t -> resolveTableOrJoin(t).stream()).toList());
-			OngoingReadingWithWhere m2 = (x.$where() != null) ? m1.where(condition(x.$where()))
-					: (OngoingReadingWithWhere) m1;
+			OngoingReading m2;
```

Comment:
I know, it's private and deep down in the code..but `m1` and `m2` does not really click for naming, or? Same would go with `match1` and `match2` because as far as I can tell, one is `matchWithCondition` and one `matchWithoutCondition`, right?